recipes/analisi/meta.yaml

```diff
@@ -0,0 +1,72 @@
+{% set name = "analisi" %}
+{% set version = "0.1.8" %}
+{% set build = 0 %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/rikigigi/analisi/archive/v{{ version }}.tar.gz
+  sha256: 13a1f025723e96d84a354761b4bcc4afdad30ef5abc3397e63792410efa7fa6d
+
+build:
+  number: {{ build }}
+  skip: True  # [not linux]
+  string: py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ mpi }}_{{ build }}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - make
+    - python
+  host:
+    - python
+    - {{ mpi }}
+  run:
+    - python
+    - {{ mpi }}
+
+test:
+  imports:
+    - pyanalisi
+  commands:
+    - |
+
+    - export OMPI_MCA_plm=isolated                          # [mpi == "openmpi"]
+    - export OMPI_MCA_btl_vader_single_copy_mechanism=none  # [mpi == "openmpi"]
+    - export OMPI_MCA_rmaps_base_oversubscribe=yes          # [mpi == "openmpi"]
+    - export OMPI_ALLOW_RUN_AS_ROOT=1                       # [mpi == "openmpi"]
+    - export OMPI_ALLOW_RUN_AS_ROOT_CONFIRM=1               # [mpi == "openmpi"]
+
+    - analisi_serial
+    - mpiexec -n 1 analisi
+about:
+  home: https://github.com/rikigigi/analisi
+  license: GPL-2.0-or-later
+  license_file:
+     - LICENSE
+     - boost/LICENSE_1_0.txt
+     - eigen-eigen-323c052e1731/COPYING.BSD
+     - eigen-eigen-323c052e1731/COPYING.GPL
+     - eigen-eigen-323c052e1731/COPYING.LGPL
+     - eigen-eigen-323c052e1731/COPYING.MINPACK
+     - eigen-eigen-323c052e1731/COPYING.MPL2
+     - eigen-eigen-323c052e1731/COPYING.README
+     - fftw-3.3.8/COPYING
```

Comment:
Is there a reason why you can't depend on `boost-cpp`, `eigen` and `fftw` instead of vendoring them?

Reply:
This is to avoid a lot of headaches when compiling from source in clusters where those dependencies are not installed (or broken) and the user is not an expert in compiling dependencies from source, boost in particular. There is no problem to use the system ones, if the environment is healthy. I would add this as an option to be used only in conda to build the package, keeping the folders with the (unused) libraries in the released tarball to cope with the broken cluster compilation problem. Tell me what you think!

Reply:
That sounds exactly like the "right" way to do this. 👍